sktime/forecasting/conditional_invertible_neural_network.py

```diff
@@ -208,7 +208,19 @@ def _fit(self, y, fh, X=None):
         Returns
         -------
         self : reference to self
+        Common Errors:
+        --------------
+        1. If `window_size` > len(y_train), an error will be raised.
+        2. If curve fitting fails, ensure your data and window_size are suitable.
+
         """
+        if self.window_size > len(y):
+            raise ValueError(
+                f"Invalid window_size: {self.window_size}. "
+                f"It must be less than or equal to the size of the training data "
```

Comment:
```suggestion
                "It must be less than or equal to the size of the training data "
```

This does not need to be f-string.

Reply:
yeah i am on it